released/packages/coq-mathcomp-finmap/coq-mathcomp-finmap.2.1.0/opam

```diff
@@ -17,8 +17,9 @@ which will be used to subsume notations for finite sets, eventually."""
 build: [make "-j%{jobs}%"]
 install: [make "install"]
 depends: [
-  "coq" { (>= "8.16" & < "9.1~") | (= "dev") }
-  "coq-mathcomp-ssreflect" { (>= "2.0.0" & < "2.4~") | (= "dev") }
+  ("coq" {>= "8.20" & < "8.21~"}
+  | "rocq-core" {>= "9.0" | = "dev"})
```

Comment:
```suggestion
  "coq" { (>= "8.16" & < "9.1~") | (= "dev") }
```
I don't understand why `rocq-core` should show up here. This is a Coq package that happens to be compatible with the `coq` shim for Rocq 9. I don't think we should change the probably hundreds of packages that are compatible with the `coq` shim for Rocq 9.

Reply:
Given that the `Makefile` explicitly invokes `coq_makefile`, I would even say that it is a mistake to depend on `rocq-core`.

Reply:
OK, that explains my build issues, thanks! I'll work towards a solution.
Meanwhile, finmap got a new release tagged (https://github.com/math-comp/finmap/releases/tag/2.2.0), but I think we should wait until the dependencies are sorted to publish to opam.
I propose making finmap 2.1.0 and 2.2.0 compatible with Rocq 9 through the Coq shim, then to publish a 2.2.1 tag after fixing the build script on finmap.

Reply:
@hoheinzollern what is the issue with just applying my suggestion? This allows finmap to be install with MC 2.4.0, on whichever Coq versions it supports.

Reply:
Yes, let's do that, we will need a new point release of finmap with a fixed build, so one can install it without depending on the coq shim if necessary.